src/Context/index.jsx

```diff
@@ -48,9 +49,10 @@ export const ContextWrapper = props => {
         `users/${userId}/trainings/${paramsId}`,
         dataEdit
       );
-      Notification(messages);
+      Notification(messages, "success");
     } catch (error) {
-      console.log(error);
+      console.log(12222, error);
```

Comment:
remove log

Reply:
thankyou for review, this issue already fix on 4da229e93e8066f1e4b3e33ea8e2a40e32f9c6d0